src/style.css

```diff
@@ -0,0 +1,131 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.product__card {
+  color: #060b35;
+
+  margin: 10px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  display: flex;
+  justify-content: center;
+
+  margin-bottom: 40px;
+}
+
+.product__img {
+  margin-top: 32px;
+  width: 160px;
+}
+
+.product__info {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  margin: 0 16px;
```

Comment:
Don't use margin-left and margin-right properties for all the elements
Just set correct paddings to the card